src/sql/magic.py

```diff
@@ -408,11 +408,14 @@ def interactive_execute_wrapper(**kwargs):
         # JA: added DatabaseError for MySQL
         except (ProgrammingError, OperationalError, DatabaseError) as e:
             # Sqlite apparently return all errors as OperationalError :/
-
             if self.short_errors:
                 print(e)
             else:
                 raise
+            print(
```

Comment:
We can utilize the COMMUNITY String in `ploomber_core/exception.py`, https://github.com/ploomber/core/blob/main/src/ploomber_core/exceptions.py#L12

Probably 
```
from ploomber_core import exceptions
...
print (exceptions. COMMUNITY)
...
```